src/components/common.ts

```diff
@@ -151,14 +151,37 @@ export interface NonHTMLAnimatedProps<A extends string, T extends HTMLTags = 'di
     /**
      * The mapping of animationName A to an AnimationInput
      */
-    animations?: Record<A, AnimationInput>;
+    animations: Record<A, AnimationInput | null>;
```

Comment:
Made animations required but allow individuals to be null. Wouldn't ever make sense for this prop to be missing and still use the component